### DISCUSSION TITLE
Multichannel (7.1.2) version not recognized in Logic Pro X

### DISCUSSION THREAD

**Discussion by djniviro** [apohl79/audiogridder#1199] [Help]
As written in the title, the Agridder plugin is not recognized in Logic Pro's 7.1.2 Dolby Atmos configuration. However both stereo, mono and multi-mono are available.

Is there any chance to build in the option to run the plugin natively as a 7.1.2 plugin?

**Comment by apohl79** [comment]
Ok thx, will have to check this out!
